desktop/cmp/grid/editors/SelectEditor.ts

```diff
@@ -19,12 +19,19 @@ export const [SelectEditor, selectEditor] = hoistCmp.withFactory<SelectEditorPro
     memo: false,
     observer: false,
     render(props, ref) {
+        const quickEdit = !props.gridModel.fullRowEditing && !props.inputProps.enableMulti;
+
         props = {
             ...props,
             inputProps: {
                 hideDropdownIndicator: true,
                 hideSelectedOptionCheck: true,
                 selectOnFocus: false,
+                onCommit: () => {
```

Comment:
just to be clear, if the application supplies an onCommit of its own, that will stomp on this behavior.  Maybe that 's ok, or maybe we want to rework this, so we always call both?

Reply:
I'd say that's tolerable - I don't imagine that would be common, and you could argue it falls under the general umbrella of "overriding some inputProps might break or change built-in behaviors" with these inputs.

Since it's non critical and is in the right direction (allowing app to override hoist) probably simplest to let it be.  If an app dev wonders why they aren't getting this subtle improvement, we can look here and know what to do :)